### ISSUE TITLE
Shpw correlation matrix of fixed effects 

### ISSUE THREAD

**Issue by iago-pssjd** [glmmTMB/glmmTMB#557]
It would be nice that correlation matrix of fixed effects was shown as it is in `lme4::glmer.`

**Comment by bbolker** [comment]
You mean in the summary output, right?  This wouldn't be too hard to implement, although I would be tempted to make the default FALSE. FWIW you can get this information (although not with formatting as nice as in `lme4::print.summary.merMod()` via `cov2cor(vcov(fitted_model)$cond)` ...

**Comment by iago-pssjd** [comment]
Yes, I meant in the summary output. For me it would be enough if we have the option of add it even if the default is FALSE.  
Thank you for the instruction to get it.

**Comment by mmaechler** [comment]
It's a matter of taste.  I do remember we had some thoughtful and fruitful chats (in Banff back then) about getting to a smart default behaviour on the current  `print.summary.merMod()`  in `lme4`.
I've always felt a bit proud about the solution we had converged at with `lme4` .. and so my vote would've been to do exactly the same in `glmmTMB`.
